src/DynamoCore/packages.config

```diff
@@ -2,6 +2,6 @@
 <packages>
   <package id="Greg" version="1.0.6176.18754" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
-  <package id="RestSharp" version="105.2.3" targetFramework="net45" />
+  <package id="RestSharp" version="105.2.3" targetFramework="net45" requireReinstallation="true" />
```

Comment:
"NuGeT is automatically flagging packages which conflict with your project target or version" - interesting I never knew about this.  Is there a reason this is required for RestSharp?

Reply:
I think it has to do with how restSharp used to have multiple targets but then moved them all to a net45 folder. But I'm not sure.

Reply:
This might require some more testing to make sure that this reference is getting called at runtime correctly. At build time - it appears fine.

Reply:
please don't merge before I fix this - I would like to reinstall so this dep gets moved to the net 4.6 target.